code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -396,16 +401,19 @@
 	explanation_text = "Start a collection of corpses. Don't kill people to get these corpses."
 
 /datum/objective/abductee/floors
-	explanation_text = "Replace all the floor tiles with carpeting, wooden boards, or grass."
+	explanation_text = "Replace all the floor tiles with carpeting, grass or bling."
 
 /datum/objective/abductee/POWERUNLIMITED
 	explanation_text = "Flood the station's powernet with as much electricity as you can."
 
 /datum/objective/abductee/pristine
-	explanation_text = "Ensure the station is in absolutely pristine condition."
+	explanation_text = "The CEO of Nanotrasen is coming! Ensure the station is in absolutely pristine condition."
+
+/datum/objective/abductee/nowalls
+	explanation_text = "The crew must get to know one another better. Break down the walls inside the station!"
 
-/datum/objective/abductee/window
-	explanation_text = "Replace all normal windows with reinforced windows."
+/datum/objective/abductee/wall
+	explanation_text = "There are too many new crew members coming to this station. You must build a wall!"
```

Comment:
I feel like this will just lead to them walling off arrivals which seems sorta griefy